### PR TITLE
this causes me pain

### PR DESCRIPTION
**Pull request by Piplup7575**
woooo grammar am i right

### CODE REVIEW

**Comment by proudparrot2** [comment]
real

**Review by zcraftelite9495 — APPROVED** [review]
These are valid spelling fixes

File: src/options.html [hunk 1]
```diff
@@ -61,13 +61,13 @@ <h1 style='color:black;'>Contributers</h1>
             <a href="#" title="Close" class="modal-close">Close</a>
             <h1 style='color:black;'>Acknowledgements</h1>
             <div style='color:black;'>
-               Acknowledgements are resources Nebula is using/has used.
```
Comment (wearrrrr) [inline src/options.html:64]: Looks good to me but this could maybe just be shortened to "Acknowledgements" instead of rewriting the line.

**Review by wearrrrr — APPROVED** [review]
Overall good changes, LGTM.

**Pull request closed without merging**